packages/helpers/src/dom/isDescendant.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-cond-assign */
+export default function isDescendant(el, attr, value) {
```

Comment:
This level of abstraction is neither here nor there IMO. Not feeling strong but if putting this into helpers, might as well do
```js
export default function isDescendant(el, callback) {
  let isChild = callback(el);
  while (!isChild && (el = el.parentNode)) {
    isChild = callback(el);
  }
  return isChild;
}

isDescendant(document.activeElement, (el) => el.getAttribute("data-pka-anchor") === "popover.content");
```
(the name could use some improvement too)